library/common-library/templates/_gke-autopilot.tpl

```diff
@@ -0,0 +1,8 @@
+{{/*
+Returns gkeAutopilot
+*/}}
+{{- define "newrelic.common.gkeAutopilot" -}}
+{{- if (or .Values.global.gkeAutopilot .Values.gkeAutopilot) -}}
+{{- default .Values.global.gkeAutopilot .Values.gkeAutopilot -}}
```

Comment:
I've been playing around a bit, and i think this is best updated to below.  I haven't commit it yet cause my unit tests fail due to nil values.  

```
{{- define "newrelic.common.gkeAutopilot" -}}
{{- coalesce .Values.global.gkeAutopilot .Values.gkeAutopilot }}
{{- end -}}
```